pkg/nodejs/prepare/application.go

```diff
@@ -20,6 +20,7 @@ type AuroraApplication struct {
 type NodeJSApplication struct {
 	Main    string `json:"main"`
 	Waf     string `json:"waf"`
+	SPA     bool   `json:"spa"`
```

Comment:
Navnet på dette feltet kan diskuteres

Reply:
Har du testet hva som skjer hvis det legges inn en string, ikke en boolean?

Reply:
@nixx får en unmarshal feil, hva tenker du?
`"Failed to build image: json: cannot unmarshal string into Go struct field NodeJSApplication.spa of type bool`